client/lib/analytics/index.js

```diff
@@ -344,6 +357,28 @@ const analytics = {
 		}
 	},
 
+	// Lucky Orange tracking
+	luckyOrange: {
+		initialized: false,
+
+		initialize: function() {
+			if ( ! analytics.luckyOrange.initialized ) {
+				if ( config( 'lucky_orange_enabled' ) && analytics.userAllowsTracking() ) {
+					const wa = document.createElement( 'script' );
+					const s = document.getElementsByTagName( 'script' )[ 0 ];
+
+					window.__lo_site_id = 77942;
+					wa.type = 'text/javascript';
+					wa.async = true;
+					wa.src = 'https://d10lpsik1i8c69.cloudfront.net/w.js';
+					s.parentNode.insertBefore( wa, s );
+
+					analytics.luckyOrange.initialized = true;
+				}
+			}
+		},
+	},
+
```

Comment:
I wonder if there are ways to modularize this code more and to remove the deep nesting. note that we do tend to prefer an early-abort over a nested block:

```js
if ( analytics.luckyOrange.initialized || ! config( 'lucky_orange_enabled' ) || ! analytics.userAllowsTracking() ) {
	return;
}

const wa = …
```